docker-compose-pol-3.7.0.yml

```
@@ -0,0 +1,67 @@
+    ports:
+      - "4000:4000"
+      - services:
+          web:
+            build: .
+            environment:
+              NODE_ENV:
+              MONGODB_URI: mongodb://mongo:27017/nodegoat
+            command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
+            ports:
+              - "4000:4000"
+services:
```

YAMLlint (1.37.1)

[error] duplication of key "services" in mapping

(key-duplicates)